src/main/resources/application.yaml

```diff
@@ -132,7 +132,7 @@ hapi:
     ### This enables the swagger-ui at /fhir/swagger-ui/index.html as well as the /fhir/api-docs (see https://hapifhir.io/hapi-fhir/docs/server_plain/openapi.html)
     openapi_enabled: true
     ### This is the FHIR version. Choose between, DSTU2, DSTU3, R4 or R5
-    fhir_version: R4
+    fhir_version: R5
```

Comment:
We will probably change these in profiles (as it is done in the branch for CQL) not in the global file. R4 still is more used and makes sense here.